docs/04-tutorial.md

```diff
@@ -141,15 +141,16 @@ npm i -D rollup
 Now you can run it inside the root folder of your project like so:
 
 ```console
-./node_modules/.bin/rollup --config
+npm run rollup --config
```

Comment:
This is not how it works for npm, I assume this is a misunderstanding (and I would seriously recommend to try it out yourself on your machine). `npm run` (which is short form of `npm run-script`) will only run scripts defined in your package.json file but not files in your `.bin` folder, see https://docs.npmjs.com/cli/run-script.html

To run files from `node_modules/bin`, `npx` is as far as I know the best choice. What many find confusing is that if you forgot to install the file, then `npx` will download it for you. But a local file will always have precedence. For some reason, npm fails to properly document this on their home page but there is a nice intro in their blog: https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner

Reply:
`npx rollup --config` will work though.